src/CashuMint.ts

```diff
@@ -48,7 +48,28 @@ class CashuMint {
 		customRequest?: typeof request
 	): Promise<GetInfoResponse> {
 		const requestInstance = customRequest || request;
-		return requestInstance<GetInfoResponse>({ endpoint: joinUrls(mintUrl, '/v1/info') });
+		const data = await requestInstance<GetInfoResponse>({
+			endpoint: joinUrls(mintUrl, '/v1/info')
+		});
+		// BEGIN DEPRECATED
+		// Monkey patch old contact field ["email", "me@mail.com"] Array<[string, string]>; to new contact field [{method: "email", info: "me@mail.com"}] Array<MintContactInfo>
```

Comment:
This comment could be removed and the part of the function that handles the deprecated info parsing extracted in a function called `handleDeprecatedInfo`, that way code would be documenting itself. However I am unsure about this repos policies about code comments. cc @gandlafbtc

Reply:
Done